onnx_caffe2/helper.py

```diff
@@ -12,6 +12,12 @@
 from onnx_caffe2.workspace import Workspace
 
 
+def dummy_name(seed=[0]):
```

Comment:
It's not a `seed`; there is no RNG involved.

Reply:
Also, I'm really not a fan of allocating mutable state in the default parameter of a function.